cloud/cluster/memory/state.go

```diff
@@ -0,0 +1,31 @@
+package memory
+
+import (
+	"github.com/scootdev/scoot/cloud/cluster"
+)
+
+// State holds the state of a Cluster (which is just its members)
+type State struct {
```

Comment:
I'm not sure why this deserves its own file. I'd prefer to see it folded into cluster.go